### ISSUE TITLE
Monster HP and type is inaccurate on all other devices after restarting scenario

### ISSUE THREAD

**Issue by itemforty** [Lurkars/gloomhavensecretariat#80]
When connected to a multiplayer session on the public server, and monster standees are already populated for a scenario and randomized, if _User X_ chooses **Set Scenario** >> **Restart Scenario**, monster standee numbers will change, but some of their numbers and type are inaccurate for all other users. 

User X before restarting scenario: 

![image](https://user-images.githubusercontent.com/175024/214142122-61a3f7de-cc34-419e-9a83-8f38884d33a4.png)

User X after: 

![image](https://user-images.githubusercontent.com/175024/214142163-066927ab-b62c-4acf-be9c-b2ef27cd74dd.png)

Before the restart, Valrath Tracker #2 was an elite with 14 HP. After, it was accurately switched to normal with 8 HP. 

User Y before: 

![image](https://user-images.githubusercontent.com/175024/214142227-9289fe29-aaba-43ca-8281-5de5b26772cc.png)

User Y after: 

![image](https://user-images.githubusercontent.com/175024/214142749-dc774693-42df-4a8d-b15c-640ffc341dcf.png)

After restarting, User Y's client thinks #2 is an Elite with 8 HP. 

I was able to replicate this more than a dozen scenarios in Forgotten Circles and Gloomhaven. As far as I can tell it only happens when the scenario is restarted using **Set Scenario** >> **Restart Scenario**, so the easy workaround is to choose a different random scenario and then choose the right one again.

**Comment by Lurkars** [comment]
Thanks for reporting, interesting that just noticed yet. Must be an old bug, where I didn't check for monster type. Should be fixed in [v0.48.10](https://github.com/Lurkars/gloomhavensecretariat/releases/tag/v0.42.10).

**Comment by Lurkars** [comment]
Please verify fix and close issue if solved.

**Comment by itemforty** [comment]
Further investigation, I setup a desktop client, picked scenario 109, and let it sit for a while. Here are the Valrath Tracker standee numbers on the "Host" system:

![image](https://user-images.githubusercontent.com/175024/214151861-48eb02bc-8545-403a-85fe-586fd67d8e25.png)

I then had my players open GHS, which was already on 109, and connect to the server. Player 1:

![image](https://user-images.githubusercontent.com/175024/214152910-6f5c7fd9-2725-4199-839d-4bed171373fb.png)

Player 2:

![image](https://user-images.githubusercontent.com/175024/214153063-5764c6ea-4aa7-45cd-b017-e0d2570d1156.png)

So, to replicate:

1. Open Client X up to scenario 109 (good example because of number of monsters) and connect to server.
2. Open Client Y up to scenario 109 as well BEFORE connecting to server. 
3. Connect Client Y to same server as Client X. 

I started to track this down because this happened when we started the scenario. 109 is a really dense map, so when we got to the point where we're matching app standee numbers to the physical standee types, we thought maybe we had mixed up something until we noticed it was only incorrect for one client. We switched to a different app unfortunately, so we didn't get a chance to test this.
